Do not execute unit-tests, will fail in cross compilation.

--- bmake/boot-strap.orig	2014-02-17 14:39:17.600113770 +0100
+++ bmake/boot-strap	2014-02-17 14:39:32.262930460 +0100
@@ -407,9 +407,6 @@ op_build() {
 	[ -s make-bootstrap.sh ] || op_configure
 	chmod 755 make-bootstrap.sh || exit 1
 	./make-bootstrap.sh || exit 1
-	case "$op" in
-	build) op_test;;
-	esac
 }
 
 op_test() {
@@ -428,7 +425,6 @@ op_clean() {
 }
 
 op_install() {
-	op_test
 	case "$INSTALL_PREFIX,$INSTALL_BIN,$prefix" in
 	,$HOST_TARGET/bin,*/$HOST_TARGET)
 		INSTALL_PREFIX=`dirname $prefix`
@@ -443,7 +439,6 @@ op_all() {
 	if [ -n "$INSTALL_PREFIX" ]; then
 		op_install
 	else
-		op_test
 		MAKE_VERSION=`sed -n '/^MAKE_VERSION/ { s,.*=  *,,;p; }' $srcdir/Makefile`
 		echo You can install by running:
 		echo
